scripts/zones/Beadeaux_[S]/mobs/Da_Dha_Hundredmask.lua

```diff
@@ -1,7 +1,7 @@
 -----------------------------------
 -- Area: Beadeaux [S] (92)
 --  NM:  Da'Dha Hundredmask
--- @spawnmob 17154195
+-- @spawnmob 17154095
```

Comment:
okay awesome, ill be fixing small things like this for the next few weeks as I got a GM spot on Era <3